3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1796,6 +1796,55 @@ Questi ultimi dovranno agire di conseguenza risolvendo i problemi emersi o piani
 
 Per avere traccia degli elementi verificati che costituiscono il prodotto, su GitHub è possibile visualizzare l'insieme delle pull request approvate ed integrate mediante merge.
 
+
+== Processo di Transizione <processo_transizione>
+Lo scopo del processo di Transizione è di stabilire la capacità di un sistema di fornire servizi specificati dalle richieste del Proponente all'interno di un'ambiente operativo differente da quello di sviluppo.
+
+=== Risultati
+Come risultato della corretta applicazione del processo di transizione, otteniamo:
+
+- il #man viene redatto;
+- gli utenti vengono supportati e allenati all'utilizzo del sistema;
+- viene rilasciata una versione del sistema;
+- viene effettuato il deploy su Azure.
+
+=== Attività
+Questo processo è composto da varie attività indipendenti e di diversa natura.
+
+Le seguenti attività non sono quindi necessariamente consequenziali, ma la loro corretta applicazione porta il conseguimento dei risultati precedentemente descritti.
+
+In particolare descriviamo:
+
+==== Rilascio dell'ultima versione del sistema
+La repository Github dedicata a WMS3 si compone di un branch _dev_ necessario per gli sviluppatori, in quanto fonte da cui vengono creati i feature branch necessari al conseguimento degli obiettivi di una task e destinazione dei merge a seguito delle successive pull request. Viene quindi definito un altro branch _release_, orfano e luogo in cui vengono effettuate i rilasci del sistema.
```

Comment:
- ~~Github~~ -> GitHub;
- nomi dei branch in `monospace` per coerenza con il resto del documento.